packages/lesswrong/server/collections/ultraFeedEvents/collection.ts

```diff
@@ -20,6 +20,12 @@ export const UltraFeedEvents: UltraFeedEventsCollection = createCollection({
       WHERE "collectionName" = 'Comments' AND "eventType" = 'served';
     `);
     
+    indexSet.addCustomPgIndex(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "ultraFeedEvents_userId_feedItemId_viewed_idx"
```

Comment:
Are there event types other than `viewed` and `served`?  Am worried about the name of the index while the `WHERE` clause checks `eventType" != 'served'`

Reply:
const ALLOWED_EVENT_TYPES = ["served", "viewed", "expanded", "interacted", "seeLess"];

Reply:
ok, renamed